decidim-core/app/cells/decidim/author_box/show.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card__author">
```

Comment:
Not sure about this. The original layout was extracted from the proposals `show.html.erb` view, which didn't have this class. The M card did have this class, that's why I this class appeared in the `ProposalM` cell.

Can you double check this?

Reply:
I'll check it

Reply:
Yes, you're right

https://github.com/decidim/decidim/blob/08043eea7efc9ec9b3eb7c26d35eb50667900dd3/decidim-proposals/app/views/decidim/proposals/proposals/_proposal.html.erb#L8

https://github.com/decidim/decidim/blob/08043eea7efc9ec9b3eb7c26d35eb50667900dd3/decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb#L10

Reply:
Good catch then! :muscle: You'll have to partially undo the changes introduced by 08d380aa988c90bafc4e5785ade76521b2846e6a then :smile: